### PR TITLE
Add quic support, part 1

### PR DESCRIPTION
**Pull request by qzhuyan**
Basic quic support, one stream per connection.

removed support for outdated OS version such like centos6 and ubuntu 16.04

note, I have chicken/egg problem here. Without quic support in emqx, no meaningful tests can be added here.

### CODE REVIEW

**Comment by qzhuyan** [comment]
worth to mention that I steal some ci code from @x1001100011  in https://github.com/emqx/emqtt/pull/128 ❤️

File: Makefile [hunk 1]
```diff
@@ -4,18 +4,22 @@ CT_NODE_NAME = ct@127.0.0.1
 
 REBAR := $(CURDIR)/rebar3
 
-REBAR_URL := https://s3.amazonaws.com/rebar3/rebar3
+#REBAR_URL := https://s3.amazonaws.com/rebar3/rebar3
```
Comment (k32) [inline Makefile:7]: Forgotten change?

**Comment by k32** [comment]
LGTM apart from the makefile changes

File: Makefile (hunk 1 above)
Comment (qzhuyan) [reply Makefile]: do it properly in latest commit, thanks

**Review by k32 — APPROVED** [review]

**Pull request merged**